### ISSUE TITLE
apply_transform missing type annotations in 4 label transforms

### ISSUE THREAD

**Issue by rhysmcneill** [fepegar/torchio#1452]
### Bug summary

Four label transforms override `apply_transform` without type annotations on the `subject` parameter or return type. Every other transform in the codebase uses `subject: Subject) -> Subject:`, so these four are inconsistent and the type checker (`ty`) cannot verify their signatures.

### Code for reproduction

```python
import inspect
import torchio as tio

for cls in [tio.Contour, tio.OneHot, tio.RemapLabels, tio.SequentialLabels]:
    sig = inspect.signature(cls.apply_transform)
    print(cls.__name__, sig)
```

### Actual outcome

```
Contour (self, subject)
OneHot (self, subject)
RemapLabels (self, subject)
SequentialLabels (self, subject)
```

No annotations on `subject` or the return type.

### Expected outcome

Consistent with all other transforms, e.g. `Resample.apply_transform`:

```
(self, subject: Subject) -> Subject
```

### Error messages

No runtime error — this is a type-consistency issue caught by static analysis.

### System info

N/A — this is a source-level inconsistency, not environment-dependent.

**Comment by fepegar** [comment]
Thanks for reporting. Would you like to submit a pull request?

**Comment by rhysmcneill** [comment]
Hey @fepegar! 

Already done, #1453. Just waiting for the CI checks then its ready to merge.
